test/extended/cli/explain.go

```diff
@@ -600,14 +590,25 @@ var _ = g.Describe("[sig-cli] oc explain", func() {
 				gvr := st.gv.WithResource(resource[0])
 				exist, err := exutil.DoesApiResourceExist(oc.AdminConfig(), resource[0], groupName)
 				o.Expect(err).NotTo(o.HaveOccurred())
-				if !exist {
+				if !exist && groupName == "security.openshift.io" {
```

Comment:
Skipping this group skips testing of `rangeallocations.range` resources which is not tested in the new cycle for `microShiftTypes`.

Reply:
If we skip when the API type is not present, we shouldn't need separate lists of types for MicroShift an OpenShift any more at all.

Reply:
It's not skipping for individual resources, but rather skipping the entire group if any of it's resource does not exist. Having separate lists of types for MicroShift and OpenShift also is not helping because the tests are unaware of the cluster type, and it will check both lists of types.